memor/template.py

```diff
@@ -42,7 +42,7 @@ def __init__(
         :return: None
         """
         self._content = None
-        self._title = None
+        self._title = "unknown"
```

Comment:
I like `None` better. It's more compatible with integration. Imagine someone wanted to save them in a data frame with a column named `title` then having some rows as None are easier to detect by NaN detectors and converting them into something better than having them as unknown.
For saving I'm also fine with that field being `null` when jsonified.

Reply:
fixed in https://github.com/openscilab/memor/pull/38/commits/a5cb7034a16e33c7fdb63f9cb341ecc2fd2fbe56